src/SFA.DAS.EmployerCommitmentsV2.Web/Controllers/DraftApprenticeshipController.cs

```diff
@@ -133,12 +125,39 @@ public async Task<IActionResult> DeleteDraftApprenticeship(DeleteDraftApprentice
                 var request = await _modelMapper.Map<DeleteDraftApprenticeshipRequest>(model);
                 await _commitmentsApiClient.DeleteDraftApprenticeship(model.CohortId.Value, model.DraftApprenticeshipId.Value, request);
                 TempData.AddFlashMessage(ApprenticeDeletedMessage, ITempDataDictionaryExtensions.FlashMessageLevel.Success);
-                return RedirectToAction("Details", "Cohort", new {model.AccountHashedId, model.CohortReference});
+                return await RedirectToCohortDetailsOrBingoPage(model);
             }
 
             return RedirectToOriginForDelete(model.Origin, model.AccountHashedId, model.CohortReference, model.DraftApprenticeshipHashedId);
         }
 
+        private async Task<IActionResult> RedirectToCohortDetailsOrBingoPage(DeleteDraftApprenticeshipViewModel model)
```

Comment:
Minor, but could we replace "BingoPage" with "CohortsPage" as bingo is not a thing. :-)